columnflow/tasks/framework/mixins.py

```diff
@@ -10,13 +10,12 @@
 import time
 import itertools
 from collections import Counter
-from typing import Iterable
 
 import luigi
 import law
 import order as od
 
-from columnflow.types import Sequence, Any
+from columnflow.types import Sequence, Any, Dict, Iterable, Tuple, Union, List
```

Comment:
Why are we using these classes (`Dict`, `Tuple`, `List`) for typing instead of the default python types

Reply:
I think there was an issue with the typing not too long ago, which prompted us to centralize the type definitions in cf (see PR #305 ). I simply follow this approach here, though this needs to be followed-up on for other modules as far as I'm aware.